src/components/calcite-popover/calcite-popover.scss

```diff
@@ -54,8 +54,7 @@ $image_max_height: 200px;
   }
 }
 
-:host([dir="rtl"]),
-:host-context([dir="rtl"]) {
+:host([dir="rtl"]) {
   .close-button {
```

Comment:
this one is needed I'm pretty sure.

Reply:
It won't work in safari either. Probably the best thing to do would be to look up the `dir` at render and use a class or something.

Reply:
To explain further, `dir=rtl` is not added automatically to every popover, so when it's `host-context` it will apply those styles even if only a distant ancestor is `dir=rtl`. If you use `:host` and it's not on the host, it won't apply. `:host-context` alone does not work in safari.

Reply:
@paulcpederson updated. Let me know if this is good.